Contents/Code/__init__.py

```diff
@@ -31,78 +31,73 @@ def json_decode(output):
 THREAD_MAX = 20
 
 intl_sites={
```

Comment:
I think `intl_sites` could be removed and a search in `sites_langs` used instead?